predict.py

```diff
@@ -12,7 +12,7 @@ def predict_image(
     device: str = typer.Argument("cpu", help="use cuda if your device has cuda", show_default=True)
 ):
     
-    predictor = ImageRecognition(model_path=model_path, device=device)
+    predictor = ImageRecognition(model_path=model_path, manifest_path="checkpoint/manifest.pth", device=device)
```

Comment:
this need to be donwloaded at the same time. soo, you need to make a downloader to download 2 files, you can choose to use zip or download 2 file separately,

Reply:
for the way you load it with two param it the right way I think